af/focus-ios.xliff

```diff
@@ -127,8 +139,7 @@
         <note>Label on About screen</note>
       </trans-unit>
       <trans-unit id="About.safariBullet2">
-        <source>Block Web fonts to reduce page size</source>
```

Comment:
Looks like this string just changes the casing for `Web` - so we should not do a new string ID.